test/unit/snippets.cpp

```diff
@@ -271,7 +271,7 @@ using_urls()
 
     //[snippet_quicklook_modifying_5
     params p = u.params();
-    p.emplace_at(p.find("name"), "name", "Vinnie Falco");
+    p.replace(p.find("name"), "name", "Vinnie Falco");
```

Comment:
Don't use my name, LOL.. Dont use real names. Use "John Doe" and "Jane Doe."

Reply:
LOL... I don't even remember when that was included. 

Yes. It sounds narcissistic. :smile: